cmd/main.go

```diff
@@ -28,11 +29,15 @@ func main() {
 	c, err := cli.New(
 		cli.WithCommandName("kubebuilder"),
 		cli.WithVersion(versionString()),
+		cli.WithDefaultProjectVersion(config.Version2),
```

Comment:
IMO it needs to be 3

Reply:
That will depend if `v3.0.0-alpha1` is released before or after this is merged. For now I kept the current behavior, if `v3.0.0-alpha1` is released and it swaps the defult version to `3-alpha` before this gets merged, I will update accordingly.

Reply:
Actually the default project version is 3-alpha right now:
```console
$ mkdir test
$ go mod init test && go get sigs.k8s.io/kubebuilder@master && rm go.mod go.sum
$ kubebuilder init
$ cat PROJECT 
domain: my.domain
layout: go.kubebuilder.io/v2
projectName: blah
repo: sigs.k8s.io/kubebuilder/v2
version: 3-alpha
```

Reply:
Ok, changed